addons/hr_attendance/static/src/components/attendance_menu/attendance_menu.js

```diff
@@ -0,0 +1,74 @@
+/* @odoo-module */
+
+
+import { Component, useState, onWillStart } from "@odoo/owl";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { deserializeDateTime } from "@web/core/l10n/dates";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { useDebounced } from "@web/core/utils/timing";
+const { DateTime } = luxon;
+
+export class ActivityMenu extends Component {
+    static components = {Dropdown, DropdownItem};
+    static props = [];
+    static template = "hr_attendance.attendance_menu";
+
+    setup() {
+        this.orm = useService("orm");
+        this.rpc = useService("rpc");
+        this.action = useService("action");
+        this.userId = useService("user").userId;
+        this.ui = useState(useService("ui"));
+        this.employee = false;
+        this.state = useState({checkedIn: false});
+        this.onClickSignInOut = useDebounced(this.signInOut, 200, true);
+        this.nextAction = "hr_attendance.hr_attendance_action_my_attendances";
+        onWillStart(this.onWillStart);
+
+    }
+
+    async onWillStart() {
+        await this.searchReadEmployee();
+    }
+
+    async searchReadEmployee(){
+        const result = await this.rpc("hr_attendance/attendance_user_data");
+        this.employee = result;
+        if (this.employee.id) {
+            this.hoursToday = registry.category("formatters").get("float_time")(
```

Comment:
registry.category("formatters") can be defined outside the component as a constant